### PR TITLE
fix(docs): update features.md // update changed-vanilla.mechanics.md

### PR DESCRIPTION
**Pull request by JoField08**
added snippet to util.md // included snipped in features.md and changed-vanilla-mechanics.md // added subcategory for the util

### CODE REVIEW

File: Writerside/topics/reusablecontent/util.md [hunk 1]
```diff
@@ -12,21 +13,25 @@
...
 > \
+> **Das Modpack könnt ihr
+auf [Modrinth](https://modrinth.com/modpack/castcrafter-survival-server "Klicke, um zum Modpack zu gelangen!")
+herunterladen.**
 > \
->**Änderungen am Modpack oder das Hinzufügen zusätzlicher Mods können zu Inkompatibilitäten und unter Umständen zu einem Ausschluss des Servers führen.**
->**Wir übernehmen keine Verantwortung für modifizierte Versionen des Modpacks.**
+> \
+> **Änderungen am Modpack oder das Hinzufügen zusätzlicher Mods können zu Inkompatibilitäten und unter Umständen zu
+einem Ausschluss des Servers führen.**
```
Comment (twisti-dev) [inline Writerside/topics/reusablecontent/util.md:33]: hier auch?

**Review by twisti-dev — CHANGES_REQUESTED** [review]

File: Writerside/topics/reusablecontent/util.md (hunk 1 above)
Comment (JoField08) [reply Writerside/topics/reusablecontent/util.md]: ctrl + alt + L

File: Writerside/topics/reusablecontent/util.md [hunk 2]
```diff
@@ -12,21 +13,25 @@
...
->Alle enthaltenen Mods sind erlaubt und mit dem Server kompatibel.
->\
->**Das Modpack könnt ihr auf [Modrinth](https://modrinth.com/modpack/castcrafter-survival-server "Klicke, um zum Modpack zu gelangen!") herunterladen.**
+> Um das Spielerlebnis auf dem Server zu optimieren gibt es ein offizielles Modpack, welches ausschließlich getestete
+> Modifikationen enthält.
+> Alle enthaltenen Mods sind erlaubt und mit dem Server kompatibel.
 > \
+> **Das Modpack könnt ihr
+auf [Modrinth](https://modrinth.com/modpack/castcrafter-survival-server "Klicke, um zum Modpack zu gelangen!")
+herunterladen.**
```
[the remark this answers is not in the document]
Comment (JoField08) [reply Writerside/topics/reusablecontent/util.md]: ctrl + alt + L

[the remark this answers is not in the document]
Comment (JoField08) [reply Writerside/topics/reusablecontent/util.md]: https://github.com/Keviro/community-server-docs/pull/152/commits/eec57fe91080bee48432099afe01c269066073b0

File: Writerside/topics/reusablecontent/util.md (hunk 1 above)
Comment (JoField08) [reply Writerside/topics/reusablecontent/util.md]: https://github.com/Keviro/community-server-docs/pull/152/commits/eec57fe91080bee48432099afe01c269066073b0

**Review by twisti-dev — APPROVED** [review]

**Pull request merged**